### PR TITLE
No sync on arm 

### PR DESCRIPTION
**Pull request by AlessandroAU**
For racing, adds option to skip sync packets after quad is armed. Better RC data and quicker syncs

### CODE REVIEW

**Comment by JyeSmith** [comment]
A few other random changes in here 😑

Ill test fly this on the weekend, but previously hacked together nosync code has worked well,

@JBKingdon worthwhile updating the switch packets to send the arm aux every packet when defined?

File: src/src/rx_main.cpp [hunk 1]
```diff
@@ -715,20 +723,29 @@ void loop()
         }
         else
         {
+            #ifdef NO_SYNC_ON_ARM
+            CURR_RATE_MAX = 4; //switch between 200hz, 100hz, 50hz, rates
```
Comment (JBKingdon) [inline src/src/rx_main.cpp:727]: This comment is the same as the original one - one of them must be wrong?

**Comment by JBKingdon** [comment]
> @JBKingdon worthwhile updating the switch packets to send the arm aux every packet when defined?

Ah, you mean now that the channel used for arm is configurable via 

-DARM_CHANNEL=5

Yes, that would be a good change to make. I created issue #157 to document it.

File: src/src/rx_main.cpp (hunk 1 above)
Comment (JyeSmith) [reply src/src/rx_main.cpp:727]: I think `CURR_RATE_MAX = 4` breaks 2.4 as it only has 3 air modes.

Comment (JyeSmith) [reply src/src/rx_main.cpp:728]: I see Sandro snuck a commit in to fix this https://github.com/AlessandroAU/ExpressLRS/commit/345fe0158d497a75ff5c0252e4f11e932f69e2ff

Comment (AlessandroAU) [reply src/src/rx_main.cpp:728]: Yup fixed

File: src/user_defines.txt [hunk 2]
```diff
@@ -15,7 +15,7 @@
 ### STEP1: ### 
 ### Enable one of these regulatory domains by removing a leading '#' ###
 
-#-DRegulatory_Domain_AU_915
+-DRegulatory_Domain_AU_915
```
Comment (JyeSmith) [inline src/user_defines.txt:18]: Should commented by default

**Pull request closed without merging**